.clang-tidy

```diff
@@ -24,5 +24,12 @@ abseil-*,                                               \
 -google-readability-namespace-comments,                 \
 -llvm-namespace-comment,                                \
 -modernize-use-trailing-return-type,                    \
--fuchsia-default-arguments-calls                        \
+-fuchsia-default-arguments-calls,                       \
+# TODO remove following lines,                          \
```

Comment:
Is this  real TODO before the merge?

Reply:
No, its just with those lines we can see some warnings, without, we are overflowded with warnings, better like this to start